spacy/ml/models/coref_util.py

```diff
@@ -0,0 +1,235 @@
+from thinc.types import Ints2d
+from spacy.tokens import Doc
+from typing import List, Tuple, Callable, Any, Set, Dict
+from ...util import registry
+import torch
```

Comment:
We need to avoid these "raw" imports of `torch` as it's only an optional dependency of `spacy`